js/tests/unit/tooltip.spec.js

```diff
@@ -3,7 +3,7 @@ import EventHandler from '../../src/dom/event-handler'
 import { noop } from '../../src/util/index'
 
 /** Test helpers */
-import { getFixture, clearFixture, jQueryMock, createEvent } from '../helpers/fixture'
+import { clearFixture, createEvent, getFixture, jQueryMock } from '../helpers/fixture'
```

Comment:
You must be using any code formatter in your IDE :sweat_smile:

Reply:
HAHAHA jetBrains like them sorted :yum: